api/Node.json

```diff
@@ -1111,10 +1113,12 @@
               "version_removed": "33"
             },
             "safari": {
-              "version_added": null
+              "version_added": "≤4",
```

Comment:
This and the following edits are for the attributes in https://github.com/mdn/browser-compat-data/issues/6545. The data is correct though, so I'll merge this to make it easier to get all the versions right when the cleanup suggested in that issue happens.